src/styles/style-stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: 96px;
+  height: 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars__star:nth-child(-n+4) {
```

Comment:
According this comment add other modifiers too 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/529ae1ba-d880-485f-be31-29daee7b5fe4)
```suggestion
.stars__star:nth-child(-n+4),
.stars__star:nth-child(-n+3),
.stars__star:nth-child(-n+2),  
............................................ {
```

Reply:
@loralevitska Я не розумію цієї логіки... Як це зробити?
Для чого це потрібно, якщо ми й так змінюємо кількість зірок в CSS?

Reply:
- we do this so that your code can work for all modifiers, that's how on the example screen I wrote another modifier and your code doesn't work for 5 active stars
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/7152d413-740a-4421-be16-008a26298ed9)
- you can use this code from your previous star challenge, here is your code from last month, make this decision
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/dfb9f16f-e658-4497-9796-5444918e84b1)
- correspondence in git, is considered working as one code base on the project, which is used in companies by many developers from different countries. It is generally accepted to communicate in English in the Git. Let's follow this recommendation for your practice and for further adaptation in work processes)
- for future, if you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review